recipes/jsonLookup/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "jsonLookup" %}
+{% set version = "0.7.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3f88add5f8f3d596ba817f024444eb49bb85815add7d564f46b5aaf926403b29
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - django
```

Comment:
`django` is needed at package build-time, not just to import/use it?

Reply:
removed it from the host